packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -49,6 +49,8 @@ export const messages = {
   parameter_gatherer_enter_alias_name:
     'Enter a scratch org alias or use default alias',
   parameter_gatherer_enter_project_name: 'Enter project name',
+  parameter_gatherer_paste_forceide_url: 'Paste forceide:// URL',
+  parameter_gatherer_invalid_forceide_url: 'The forceide:// URL is incorrect',
```

Comment:
@ruthemmanuelle new text when prompting for input and error in case of invalid input

Reply:
Can we give any guidance on where they get this URL and/or on why the value supplied is "incorrect"?

Reply:
There is a plan to give that guidance on the server in the Setup UI when the user creates an Apex Debug session. The screen that currently shows the drag&drop instructions will be updated to show the VS Code instructions.

We could technically give more details on what is missing in the URL. However, the URL should come as a 1:1 copy from the server page, where it's listed. Thus, it might be better to advice a user to verify that they copied the complete URL from the server page?

Reply:
@guw - A screenshot/gif usually helps a lot under such situations.

Reply:
Thanks for the GIF and info. How about these?

parameter_gatherer_paste_forceide_url: 'Paste forceide:// URL from Setup',
parameter_gatherer_invalid_forceide_url: 'The forceide:// URL is invalid. From your subscriber\'s org, copy and paste the forceide:// URL shown on the Apex Debugger page in Setup.',